apps/backend/src/resolvers/queries/CallQuery.ts

```diff
@@ -14,10 +14,10 @@ export class CallQuery {
   }
 
   @Query(() => Call, { nullable: true })
-  callByQuestionId(
-    @Arg('questionId', () => String) questionId: string,
+  getCallAllocatedTimeUnit(
```

Comment:
While we're only using this in the frontend for the time unit, as a backend query it could be used for anything on the Call object, so would something like `getCallOfAnswersProposal` be better here too?

Reply:
I agree with @ACLay. This must be frontend agnostic.